packages/loader/container-loader/src/utils.ts

```diff
@@ -152,12 +152,12 @@ export async function runWithRetry<T>(
             lastError = err;
             // If the error is throttling error, then wait for the specified time before retrying.
             // If the waitTime is not specified, then we start with retrying immediately to max of 8s.
-            retryAfter = getRetryDelayFromError(err) ?? Math.min(retryAfter * 2, 8000);
```

Comment:
Personally, I'd prefer all the times to be milliseconds everywhere, and rather rename getRetryDelayFromError and make it return milliseconds (and fix couple places).

Note - same problem exists in getFileLink.

Reply:
this is just a bug fix, rather not couple it with refactoring. added an issue: #5881